src/app/parcels/parcelsTable/sortableColumns.ts

```diff
@@ -1,6 +1,15 @@
 import { SortOptions } from "@/components/Tables/Table";
 import { ParcelsTableRow, ParcelsSortMethod } from "./types";
 
+export const defaultParcelsSort: ParcelsSortMethod = (query, sortDirection) =>
+    query
+        .order("packing_date", { ascending: sortDirection === "asc" })
+        .order("packing_slot_order")
+        .order("is_delivery", { ascending: false })
+        .order("client_is_active", { ascending: false })
+        .order("client_address_postcode")
+        .order("collection_centre_name");
```

Comment:
This slightly changes the default sort, but this is a correction - the default sort should always have been exactly the same as the packing date sort